src/ValueObjects/TransformationV2.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\TransformationMigrate\ValueObjects;
+
+class TransformationV2
+{
+    /** @var TransformationV2Block[] $blocks */
+    private $blocks;
+
+    private string $name;
+
+    private string $type;
+
+    private string $backend;
+
+    private array $inputMappingTables = [];
+
+    private array $outputMappingTables = [];
+
+    public function __construct(string $name, string $type, string $backend)
+    {
+        $this->name = $name;
+        $this->type = $type;
+        $this->backend = $backend;
+    }
+
+    public function addBlock(TransformationV2Block $block): void
+    {
+        $this->blocks[] = $block;
+    }
+
+    public function addInputMappingTable(array $inputMappingTable): void
+    {
+        $this->inputMappingTables[$inputMappingTable['source']] = $inputMappingTable;
```

Comment:
Tu moze nastat edge case, ze mas viac `rows` a maju v IM/OM rovnake tabulky s roznou konfiguraciou (napr. `destination`). Preberali ste to na calle k tejto feature? Neviem, ako by sa to dalo najlepsie osetrit. Mozno to nemusie riesit a ked bude problem, tak si to user fixne rucne.

Reply:
to může nastat a určitě tam bude minimálně na začátku potřeba uživatelská kontrola, proto se to dělá po jednotlivých bucketech a když něco takového nastane, tak to bude muset klient fixnout

Reply:
napadá mě že by se to dalo vyřešit tak že to neprojde validací, ale je to na zváženou

Reply:
zjistím to na dalším callu

Reply:
ok